docker-compose.yml

```diff
@@ -3,9 +3,11 @@ version: '3'
 services:
   web:
     build: .
-    command: bash -c "python censo/manage.py makemigrations && python censo/manage.py migrate && python censo/manage.py runserver 0.0.0.0:8000"
+    command: bash -c "python manage.py makemigrations && python manage.py migrate && python manage.py read_data --file /data/municipal_data.csv && python manage.py runserver 0.0.0.0:8000"
```

Comment:
@andreformento percebi que esse script não está totalmente correto. Ele insere todos os municípios a cada vez que é executado, logo se ele ficar no docker compose irá duplicar cada cidade na base de dados.

Talvez agora a saída pode ser remover a execução do script no Docker ou [ajustar o script](https://github.com/okfn-brasil/censo-querido-diario/blob/main/formulario/management/commands/read_data.py).

Reply:
Nossa! Nem tinha pensado sobre duplicar os dados 😱 

De qualquer forma, eu não consegui reproduzir esse comportamento. Como você fez? Eu vi que não existe nenhum volume que guarde estado da execução do banco (é sqlite local né?). Então, não consegui descobrir como simular isso via docker-compose. Teoricamente toda vez que rodar ele vai criar e popular uma base do zero.

Caso ache necessário guardar o estado do banco, por alguma questão de otimização, eu posso manter o sqlite pra guardar os dados (via volume) e quando executar novamente não precisa executara a leitura. Eu fiz [essa branch aqui](https://github.com/andreformento/censo-querido-diario/compare/fix-docker-build...andreformento:fix-docker-build-keep-state-of-database) como sugestão que já valida se tem alguma cidade cadastrada, conforme script de read que você sugeriu. Se achar válida faço o merge pra branch desse PR aqui pra vc aprovar 😄